moz_kinto_publisher/main.py

```diff
@@ -51,13 +51,23 @@ def __init__(self, filter_bucket):
     def __len__(self):
         return len(self.run_identifiers)
 
-    def is_run_valid(self, run_id):
+    def is_run_valid(self, run_id, channel):
```

Comment:
We also have this logic in `publish_crlite` - should this be a helper function?

Reply:
Yep, done!